lib/ansible/cli/__init__.py

```diff
@@ -155,6 +155,9 @@ def run(self):
         running an Ansible command.
         """
 
+        if self.options.verbosity > 1:
+            display.display(self.parser.get_version())
```

Comment:
one possible reason to check verbosity would be to make '-v' show just the config, else show the version (which also mentions the config file path albeit with a different msg)